tests/unit/CoreVideoTest.php

```diff
@@ -70,6 +77,25 @@ public function test_retrieve_core_video_attributes() {
 				}
 			}
 		';
+	}
+
+	public function test_retrieve_core_video_attributes() {
```

Comment:
Comment listing the tested attributes + `: void` return type on the method signature

Reply:
Done ✅